Building DAG of jobs...
Forcing incomplete files:
	temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.0.fimo.txt
	temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.1.fimo.txt
	temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.2.fimo.txt
	temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.3.fimo.txt
	temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.4.fimo.txt
	temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.5.fimo.txt
	temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.6.fimo.txt
	temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.7.fimo.txt
	temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.8.fimo.txt
	temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.9.fimo.txt
	temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.10.fimo.txt
	temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.11.fimo.txt
	temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.13.fimo.txt
	temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.15.fimo.txt
	temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.16.fimo.txt
	temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.18.fimo.txt
	temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.19.fimo.txt
Using shell: /usr/bin/bash
Provided cores: 40
Rules claiming more threads will be scaled down.
Job stats:
job                     count    min threads    max threads
--------------------  -------  -------------  -------------
combine_fimo_output         1              1              1
generate_tf_database        1              1              1
run_fimo                   40              1              1
split_motifs                1              1              1
total                      43              1              1

Resources before job selection: {'_cores': 40, '_nodes': 9223372036854775807}
Ready jobs (1):
	split_motifs
Select jobs to execute...
Using greedy selector because only single job has to be scheduled.
Selected jobs (1):
	split_motifs
Resources after job selection: {'_cores': 39, '_nodes': 9223372036854775806}

[Tue May 24 09:54:19 2022]
rule split_motifs:
    input: data/tf_database/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.0.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.1.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.2.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.3.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.4.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.5.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.6.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.7.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.8.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.10.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.11.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.12.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.13.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.14.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.15.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.16.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.17.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.18.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.19.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.20.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.21.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.22.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.23.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.24.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.25.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.26.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.27.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.28.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.29.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.30.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.31.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.32.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.33.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.34.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.35.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.36.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.37.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.38.txt, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.39.txt
    jobid: 4
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme
    resources: tmpdir=/tmp

[Tue May 24 09:54:20 2022]
Finished job 4.
1 of 43 steps (2%) done
Resources before job selection: {'_cores': 40, '_nodes': 9223372036854775807}
Ready jobs (40):
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
Select jobs to execute...
Selected jobs (40):
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
	run_fimo
Resources after job selection: {'_cores': 0, '_nodes': 9223372036854775767}

[Tue May 24 09:54:21 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.9.fimo.txt
    jobid: 13
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=9
    resources: tmpdir=/tmp


[Tue May 24 09:54:21 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.23.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.23.fimo.txt
    jobid: 27
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=23
    resources: tmpdir=/tmp


[Tue May 24 09:54:21 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.2.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.2.fimo.txt
    jobid: 6
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=2
    resources: tmpdir=/tmp


[Tue May 24 09:54:21 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.16.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.16.fimo.txt
    jobid: 20
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=16
    resources: tmpdir=/tmp


[Tue May 24 09:54:21 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.30.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.30.fimo.txt
    jobid: 34
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=30
    resources: tmpdir=/tmp


[Tue May 24 09:54:21 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.37.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.37.fimo.txt
    jobid: 41
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=37
    resources: tmpdir=/tmp


[Tue May 24 09:54:21 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.10.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.10.fimo.txt
    jobid: 14
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=10
    resources: tmpdir=/tmp


[Tue May 24 09:54:21 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.24.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.24.fimo.txt
    jobid: 28
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=24
    resources: tmpdir=/tmp


[Tue May 24 09:54:21 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.17.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.17.fimo.txt
    jobid: 21
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=17
    resources: tmpdir=/tmp


[Tue May 24 09:54:21 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.3.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.3.fimo.txt
    jobid: 7
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=3
    resources: tmpdir=/tmp


[Tue May 24 09:54:21 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.38.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.38.fimo.txt
    jobid: 42
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=38
    resources: tmpdir=/tmp


[Tue May 24 09:54:21 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.31.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.31.fimo.txt
    jobid: 35
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=31
    resources: tmpdir=/tmp


[Tue May 24 09:54:21 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.11.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.11.fimo.txt
    jobid: 15
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=11
    resources: tmpdir=/tmp


[Tue May 24 09:54:21 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.25.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.25.fimo.txt
    jobid: 29
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=25
    resources: tmpdir=/tmp


[Tue May 24 09:54:21 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.4.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.4.fimo.txt
    jobid: 8
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=4
    resources: tmpdir=/tmp


[Tue May 24 09:54:21 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.18.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.18.fimo.txt
    jobid: 22
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=18
    resources: tmpdir=/tmp


[Tue May 24 09:54:21 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.32.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.32.fimo.txt
    jobid: 36
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=32
    resources: tmpdir=/tmp


[Tue May 24 09:54:21 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.39.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.39.fimo.txt
    jobid: 43
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=39
    resources: tmpdir=/tmp


[Tue May 24 09:54:21 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.12.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.12.fimo.txt
    jobid: 16
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=12
    resources: tmpdir=/tmp


[Tue May 24 09:54:21 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.0.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.0.fimo.txt
    jobid: 2
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=0
    resources: tmpdir=/tmp


[Tue May 24 09:54:22 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.5.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.5.fimo.txt
    jobid: 9
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=5
    resources: tmpdir=/tmp


[Tue May 24 09:54:22 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.19.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.19.fimo.txt
    jobid: 23
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=19
    resources: tmpdir=/tmp


[Tue May 24 09:54:22 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.26.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.26.fimo.txt
    jobid: 30
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=26
    resources: tmpdir=/tmp


[Tue May 24 09:54:22 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.33.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.33.fimo.txt
    jobid: 37
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=33
    resources: tmpdir=/tmp


[Tue May 24 09:54:22 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.13.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.13.fimo.txt
    jobid: 17
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=13
    resources: tmpdir=/tmp


[Tue May 24 09:54:22 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.27.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.27.fimo.txt
    jobid: 31
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=27
    resources: tmpdir=/tmp


[Tue May 24 09:54:22 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.6.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.6.fimo.txt
    jobid: 10
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=6
    resources: tmpdir=/tmp


[Tue May 24 09:54:22 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.20.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.20.fimo.txt
    jobid: 24
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=20
    resources: tmpdir=/tmp


[Tue May 24 09:54:23 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.34.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.34.fimo.txt
    jobid: 38
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=34
    resources: tmpdir=/tmp


[Tue May 24 09:54:23 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.14.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.14.fimo.txt
    jobid: 18
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=14
    resources: tmpdir=/tmp


[Tue May 24 09:54:23 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.28.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.28.fimo.txt
    jobid: 32
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=28
    resources: tmpdir=/tmp


[Tue May 24 09:54:23 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.7.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.7.fimo.txt
    jobid: 11
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=7
    resources: tmpdir=/tmp


[Tue May 24 09:54:23 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.21.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.21.fimo.txt
    jobid: 25
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=21
    resources: tmpdir=/tmp


[Tue May 24 09:54:23 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.35.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.35.fimo.txt
    jobid: 39
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=35
    resources: tmpdir=/tmp


[Tue May 24 09:54:24 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.1.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.1.fimo.txt
    jobid: 5
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=1
    resources: tmpdir=/tmp


[Tue May 24 09:54:24 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.15.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.15.fimo.txt
    jobid: 19
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=15
    resources: tmpdir=/tmp


[Tue May 24 09:54:24 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.8.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.8.fimo.txt
    jobid: 12
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=8
    resources: tmpdir=/tmp


[Tue May 24 09:54:24 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.22.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.22.fimo.txt
    jobid: 26
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=22
    resources: tmpdir=/tmp


[Tue May 24 09:54:24 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.29.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.29.fimo.txt
    jobid: 33
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=29
    resources: tmpdir=/tmp


[Tue May 24 09:54:25 2022]
rule run_fimo:
    input: data/9mers.fa, temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.36.txt
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.36.fimo.txt
    jobid: 40
    wildcards: tf_database_file=JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme, word_length=9, i=36
    resources: tmpdir=/tmp

Terminating processes on user request, this might take some time.
Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.1.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.1.fimo.txt' died with <Signals.SIGINT: 2>.

[Tue May 24 09:57:47 2022]
Error in rule run_fimo:
    jobid: 5
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.1.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.1.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.1.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.11.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.11.fimo.txt' died with <Signals.SIGINT: 2>.

[Tue May 24 09:57:47 2022]
Error in rule run_fimo:
    jobid: 15
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.11.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.11.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.11.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.32.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.32.fimo.txt' died with <Signals.SIGINT: 2>.

[Tue May 24 09:57:47 2022]
Error in rule run_fimo:
    jobid: 36
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.32.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.32.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.32.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.3.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.3.fimo.txt' died with <Signals.SIGINT: 2>.

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.6.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.6.fimo.txt' died with <Signals.SIGINT: 2>.

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.30.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.30.fimo.txt' died with <Signals.SIGINT: 2>.

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.16.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.16.fimo.txt' died with <Signals.SIGINT: 2>.

[Tue May 24 09:57:47 2022]
Error in rule run_fimo:
    jobid: 7
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.3.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.3.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.3.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.10.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.10.fimo.txt' died with <Signals.SIGINT: 2>.

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.5.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.5.fimo.txt' died with <Signals.SIGINT: 2>.

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.25.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.25.fimo.txt' died with <Signals.SIGINT: 2>.

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.22.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.22.fimo.txt' died with <Signals.SIGINT: 2>.

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.19.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.19.fimo.txt' died with <Signals.SIGINT: 2>.

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.13.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.13.fimo.txt' died with <Signals.SIGINT: 2>.

[Tue May 24 09:57:47 2022]
[Tue May 24 09:57:47 2022]
[Tue May 24 09:57:47 2022]
Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.35.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.35.fimo.txt' died with <Signals.SIGINT: 2>.

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.24.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.24.fimo.txt' died with <Signals.SIGINT: 2>.

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.28.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.28.fimo.txt' died with <Signals.SIGINT: 2>.

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.39.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.39.fimo.txt' died with <Signals.SIGINT: 2>.

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.0.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.0.fimo.txt' died with <Signals.SIGINT: 2>.

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.18.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.18.fimo.txt' died with <Signals.SIGINT: 2>.

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.2.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.2.fimo.txt' died with <Signals.SIGINT: 2>.

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.14.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.14.fimo.txt' died with <Signals.SIGINT: 2>.

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.12.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.12.fimo.txt' died with <Signals.SIGINT: 2>.

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.3.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.3.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.3.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.3.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.32.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.32.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.32.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.32.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.11.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.11.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.11.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.11.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

[Tue May 24 09:57:47 2022]
Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.29.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.29.fimo.txt' died with <Signals.SIGINT: 2>.

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.36.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.36.fimo.txt' died with <Signals.SIGINT: 2>.

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.17.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.17.fimo.txt' died with <Signals.SIGINT: 2>.

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.4.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.4.fimo.txt' died with <Signals.SIGINT: 2>.

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.15.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.15.fimo.txt' died with <Signals.SIGINT: 2>.

[Tue May 24 09:57:47 2022]
[Tue May 24 09:57:47 2022]
[Tue May 24 09:57:47 2022]
[Tue May 24 09:57:47 2022]
[Tue May 24 09:57:47 2022]
[Tue May 24 09:57:47 2022]
Error in rule run_fimo:
    jobid: 10
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.6.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.6.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.6.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Error in rule run_fimo:
    jobid: 34
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.30.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.30.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.30.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Error in rule run_fimo:
    jobid: 20
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.16.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.16.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.16.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.3.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.3.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
[Tue May 24 09:57:47 2022]
[Tue May 24 09:57:47 2022]
[Tue May 24 09:57:47 2022]
[Tue May 24 09:57:47 2022]
[Tue May 24 09:57:47 2022]
[Tue May 24 09:57:47 2022]
[Tue May 24 09:57:47 2022]
Error in rule run_fimo:
    jobid: 14
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.10.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.10.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.10.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

[Tue May 24 09:57:47 2022]
RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.32.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.32.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
[Tue May 24 09:57:47 2022]
[Tue May 24 09:57:47 2022]
[Tue May 24 09:57:47 2022]
Error in rule run_fimo:
    jobid: 33
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.29.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.29.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.29.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

[Tue May 24 09:57:47 2022]
[Tue May 24 09:57:47 2022]
Error in rule run_fimo:
    jobid: 32
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.28.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.28.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.28.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Error in rule run_fimo:
    jobid: 9
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.5.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.5.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.5.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Error in rule run_fimo:
    jobid: 29
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.25.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.25.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.25.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Error in rule run_fimo:
    jobid: 23
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.19.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.19.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.19.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Error in rule run_fimo:
    jobid: 26
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.22.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.22.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.22.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Error in rule run_fimo:
    jobid: 17
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.13.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.13.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.13.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.6.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.6.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.6.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.6.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.30.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.30.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.30.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.30.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.16.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.16.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.16.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.16.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Error in rule run_fimo:
    jobid: 43
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.39.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.39.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.39.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Error in rule run_fimo:
    jobid: 28
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.24.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.24.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.24.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Error in rule run_fimo:
    jobid: 2
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.0.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.0.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.0.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Error in rule run_fimo:
    jobid: 22
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.18.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.18.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.18.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Error in rule run_fimo:
    jobid: 6
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.2.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.2.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.2.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Error in rule run_fimo:
    jobid: 18
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.14.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.14.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.14.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Error in rule run_fimo:
    jobid: 16
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.12.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.12.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.12.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.5.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.5.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.5.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.5.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Error in rule run_fimo:
    jobid: 39
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.35.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.35.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.35.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.10.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.10.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.10.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.10.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Error in rule run_fimo:
    jobid: 21
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.17.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.17.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.17.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Error in rule run_fimo:
    jobid: 8
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.4.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.4.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.4.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Error in rule run_fimo:
    jobid: 40
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.36.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.36.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.36.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Error in rule run_fimo:
    jobid: 19
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.15.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.15.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.15.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.11.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.11.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.28.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.28.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.28.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.28.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.29.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.29.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.29.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.29.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.23.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.23.fimo.txt' died with <Signals.SIGINT: 2>.

RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.5.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.5.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.19.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.19.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.19.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.19.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.22.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.22.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.22.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.22.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.13.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.13.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.13.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.13.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.22.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.22.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.30.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.30.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.16.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.16.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.39.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.39.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.39.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.39.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.24.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.24.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.24.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.24.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.0.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.0.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.0.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.0.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.18.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.18.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.18.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.18.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.2.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.2.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.2.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.2.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.14.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.14.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.14.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.14.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.12.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.12.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.12.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.12.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.35.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.35.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.35.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.35.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.25.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.25.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.25.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.25.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.17.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.17.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.17.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.17.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.4.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.4.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.4.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.4.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.36.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.36.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.36.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.36.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.15.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.15.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.15.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.15.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.6.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.6.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.1.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.1.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.1.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.1.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.15.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.15.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.29.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.29.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
[Tue May 24 09:57:47 2022]
RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.19.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.19.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.13.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.13.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.10.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.10.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.38.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.38.fimo.txt' died with <Signals.SIGINT: 2>.

RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.39.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.39.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.24.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.24.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.26.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.26.fimo.txt' died with <Signals.SIGINT: 2>.

RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.0.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.0.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.18.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.18.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.31.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.31.fimo.txt' died with <Signals.SIGINT: 2>.

RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.2.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.2.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.14.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.14.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.34.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.34.fimo.txt' died with <Signals.SIGINT: 2>.

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.7.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.7.fimo.txt' died with <Signals.SIGINT: 2>.

RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.35.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.35.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.21.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.21.fimo.txt' died with <Signals.SIGINT: 2>.

RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.25.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.25.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.17.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.17.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.20.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.20.fimo.txt' died with <Signals.SIGINT: 2>.

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.33.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.33.fimo.txt' died with <Signals.SIGINT: 2>.

RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.36.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.36.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.27.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.27.fimo.txt' died with <Signals.SIGINT: 2>.

RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.4.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.4.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.28.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.28.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.8.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.8.fimo.txt' died with <Signals.SIGINT: 2>.

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.9.fimo.txt' died with <Signals.SIGINT: 2>.

[Tue May 24 09:57:47 2022]
Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.37.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.37.fimo.txt' died with <Signals.SIGINT: 2>.

[Tue May 24 09:57:47 2022]
[Tue May 24 09:57:47 2022]
RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.12.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.12.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
[Tue May 24 09:57:47 2022]
[Tue May 24 09:57:47 2022]
[Tue May 24 09:57:47 2022]
[Tue May 24 09:57:47 2022]
[Tue May 24 09:57:47 2022]
[Tue May 24 09:57:47 2022]
[Tue May 24 09:57:47 2022]
Error in rule run_fimo:
    jobid: 13
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.9.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.9.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Error in rule run_fimo:
    jobid: 27
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.23.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.23.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.23.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.1.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.1.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.9.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.9.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Error in rule run_fimo:
    jobid: 30
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.26.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.26.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.26.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Error in rule run_fimo:
    jobid: 35
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.31.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.31.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.31.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

[Tue May 24 09:57:47 2022]
Error in rule run_fimo:
    jobid: 11
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.7.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.7.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.7.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Error in rule run_fimo:
    jobid: 25
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.21.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.21.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.21.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Error in rule run_fimo:
    jobid: 38
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.34.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.34.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.34.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.34.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.34.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.34.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.34.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Error in rule run_fimo:
    jobid: 31
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.27.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.27.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.27.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Error in rule run_fimo:
    jobid: 24
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.20.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.20.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.20.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Error in rule run_fimo:
    jobid: 42
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.38.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.38.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.38.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

[Tue May 24 09:57:47 2022]
Error in rule run_fimo:
    jobid: 12
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.8.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.8.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.8.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.21.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.21.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.21.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.21.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.26.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.26.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.26.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.26.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Error in rule run_fimo:
    jobid: 37
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.33.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.33.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.33.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.7.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.7.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.7.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.7.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.33.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.33.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.33.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.33.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.31.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.31.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.31.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.31.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.27.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.27.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.27.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.27.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.20.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.20.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.20.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.20.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.38.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.38.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.38.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.38.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Error in rule run_fimo:
    jobid: 41
    output: temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.37.fimo.txt
    shell:
        fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.37.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.37.fimo.txt
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.8.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.8.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.8.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.8.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.38.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.38.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.37.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.37.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.37.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.37.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

Full Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2543, in run_wrapper
    runtime_sourcecache_path,
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 80, in __rule_run_fimo
    tf_metadata_file = config['tf_metadata_file']
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/shell.py", line 296, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.23.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.23.fimo.txt' died with <Signals.SIGINT: 2>.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 617, in _callback
    raise ex
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 603, in cached_or_run
    run_func(*args)
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2557, in run_wrapper
    ex, lineno, linemaps=linemaps, snakefile=file, show_traceback=True
snakemake.exceptions.RuleException: CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.23.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.23.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo

RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.7.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.7.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.31.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.31.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.34.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.34.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.26.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.26.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.27.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.27.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.20.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.20.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.9.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.21.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.21.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.8.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.8.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.33.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.33.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.37.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.37.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
RuleException:
CalledProcessError in line 35 of /gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile:
Command 'set -euo pipefail;  fimo --skip-matched-sequence temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.23.txt data/9mers.fa > temp/JASPAR2022_CORE_vertebrates_non-redundant_pfms_meme.9mers.23.fimo.txt' died with <Signals.SIGINT: 2>.
  File "/gpfs/scratch/jtl334/thesis/pipeline/workflow/Snakefile", line 35, in __rule_run_fimo
  File "/gpfs/home/jtl334/.conda/envs/pipeline/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Complete log: .snakemake/log/2022-05-24T095418.718148.snakemake.log
unlocking
removing lock
removing lock
removed all locks
